### PR TITLE
replicasets endpoint implemented

### PR DESCRIPTION
**Pull request by bilalcaliskan**
#### Which issue(s) this PR fixes:
this pr solve issue #13  .

#### What this PR does / why we need it:
This PR creates an endpoint on the router to handle `/resources/replicasets` endpoint to serve replicasets from specified clusters.

#### Special notes for your reviewer:
Implementation done such as `/resources/deployments` endpoint implementation to use same conventional pattern. One difference is that request URL takes namespace as parameter. If it does not exists, handler lists all replicasets on all namespaces.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -76,6 +76,22 @@ func (c ClusterConnection) GetDeployments(clusters []string) (map[string][]v1.De
...
+	replicasetClusterMap := make(map[string][]v1.ReplicaSet)
+	var errors []error
+
+	for _, cluster := range clusters {
+		replicaSets, err := c.connections[cluster].AppsV1().ReplicaSets(namespace).List(metav1.ListOptions{})
+		if err != nil {
+			errors = append(errors, err)
+		}
+
+		replicasetClusterMap[cluster] = replicaSets.Items
```
Comment (mstrYoda) [inline main.go]: It be should be nice to add this line in else statement. When an error occures, no need to add empty item to map.

File: main.go [hunk 2]
```diff
@@ -266,6 +283,26 @@ func GetDeployments(w http.ResponseWriter, r *http.Request, _ httprouter.Params)
...
+	namespace := r.URL.Query().Get("namespace")
+	replicaSets, errors := clusterConnection.GetReplicaSets(strings.Split(clusters, ","), namespace)
+	if errors != nil {
+		fmt.Println(errors)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	responseBytes, err := json.Marshal(replicaSets)
+	if err != nil {
```
Comment (mstrYoda) [inline main.go:297]: Should we log this error too?

**Comment by bilalcaliskan** [comment]
good points, both done :+1:

**Review by mstrYoda — APPROVED** [review]
Thanks for your contribution

**Pull request merged**